templates/aspnet-s2i-template.json

```diff
@@ -280,7 +277,7 @@
         {
             "name": "ASP_UPSTREAM_IMAGE_TAG",
             "description": "Base OpenShift ASP S2I Image Tag",
-            "value": "latest",
+            "value": "1.0",
```

Comment:
Well... This is standard practice for OpenShift images (I think that's why we have the version number in the image name instead of as a tag), but I just read the [Kubernetes doc](https://kubernetes.io/docs/user-guide/images/) and they recommend against it:

> Note that you should avoid using :latest tag, see Best Practices for Configuration for more information.

Hopefully @bparees can help us to understand the best way to proceed here